Sprout/Models/Post.h

```diff
@@ -0,0 +1,22 @@
+//
+//  Post.h
+//  Sprout
+//
+//  Created by Sarah Wang on 7/13/21.
+//
+
+#import <Parse/Parse.h>
+#import "Goal.h"
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface Post : PFObject<PFSubclassing>
+
+@property (nonatomic, strong) NSNumber *likeCount;
```

Comment:
This is gonna be a little hand-wavey but generally I pause when I see a property with 'count' in the name because I think, "what's being counted?" and, "why not just store those?".